docs/app/pages/charts/sections/spark-charts/spark-charts/spark-charts.component.html

```diff
@@ -64,15 +63,22 @@
                 <th>Description</th>
             </tr>
             <tr>
+                <td class="attribute">theme</td>
+                <td>string</td>
+                <td>Defines the color of the bar and the track. Defaults to the <code>chart1</code> color in the
```

Comment:
Might want to add something to inform them the value they provide should be a color from our color palette